pyproject.toml

```diff
@@ -22,6 +22,33 @@ extend_skip_glob = ["**/migrations/*"]
 
 ##############################################
 
+[tool.ruff]
+target-version = "py310"
+line-length = 120
+select = ["F", "E", "B", "W", "N", "UP", "YTT", "FIX", "ASYNC", "A", "DJ", "EXE", "ICN", "G", "SLOT", "TID", "TCH", "INT", "C4", "ISC", "INP", "PIE", "RSE", "RET", "COM","PGH","FLY", "PERF", "PLE"]  # PYI,Q,SIM,PTH,PL,TRY,RUF
+ignore = [
+    "E501",                  # line-too-long: black does code formatting for us
```

Comment:
Is the `line-length` setting relevant then? (I could imagine it being relevant for auto-fixes)

Reply:
Not really. I don't know about auto-fixes, but I added `line-length` so that the formatter works (although black is still the way to go of course)

Reply:
Ah, I forgot that they want to do linting and formatting. Well, fine with me, since they claim to be black-compatible.